timeseries-streaming/timeseries-java-applications/TimeSeriesPipeline/src/main/java/com/google/dataflow/sample/timeseriesflow/verifier/TSDataPointVerifier.java

```diff
@@ -61,6 +63,11 @@ public void process(@Element TSDataPoint input, OutputReceiver<TSDataPoint> o) {
             String.format("TSDataPoint has no timestamp. %s", input.toString()));
       }
 
+      if (!(time.isAfter(GlobalWindow.TIMESTAMP_MIN_VALUE) && time.isBefore(GlobalWindow.TIMESTAMP_MAX_VALUE))) {
```

Comment:
Maybe better in a different commit?

Reply:
Sure, I reverted the commit and will apply in a separate PR.